ThumbnailSequence.cpp

```diff
@@ -1560,7 +1560,7 @@ ThumbnailSequence::CompositeItem::CompositeItem(
 bool
 ThumbnailSequence::CompositeItem::incompleteThumbnail() const
 {
-	return dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
+	return m_pThumb == 0 && dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
```

Comment:
In the meantime all versions crash on the dynamic_cast. Maybe there is some problem with the std::auto_ptr usage as it is deprecated and should be replaced by std::unique_ptr? http://stackoverflow.com/questions/3451099/stdauto-ptr-to-stdunique-ptr

I tried this but get problems in filters/page_split/Task.cpp line 208 where I get problem in passing m_ptrDbg.